File: Python/Resistor_color.py
def color_code(color):
    resistors_color = {
    "black": 0,
    "brown": 1,
    "red": 2,
    "orange": 3,
    "yellow": 4,
    "green": 5,
    "blue": 6,
    "violet": 7,
    "grey": 8,
    "white": 9,
    }
    return resistors_color[color]

def colors():
    expected = ["black", "brown", "red", "orange", "yellow", 
                "green", "blue", "violet", "grey", "white"
        ]
    return expected

File: Ruby/resistor_color.rb
class ResistorColor
    COLORS = ["black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white"]
  
    
    def self.color_code(color)
      return COLORS.find_index(color)
    end
  end

  puts ResistorColor.color_code("black")
  
  puts ResistorColor.color_code("white")
  
  puts ResistorColor.color_code("orange")
  
  puts ResistorColor::COLORS